example/WORKSPACE

```diff
@@ -86,6 +86,6 @@ conda_create(
 
 # register pythons from environment as toolchain
 register_toolchain(
-    py2_env = "py2_env",  # python2 is optional
+    # py2_env = "py2_env",  # python2 is optional
```

Comment:
I will revert this. Just FYI that it doesn't work with miniforge.

Reply:
Now I'm wondering if we should drop support for Python 2 altogether 🤔 

It's deprecated anyway and [requiring Python 3 might be a good practice](https://python3statement.org)

Reply:
I'm happy to drop it :). But maybe not in this PR. I'll leave that up to you. Let's see if tests still pass now that I'm passing `miniconda` as the default.

Do you want me to add a `miniforge` test?

Reply:
Right, dropping Python 2 should be in another pull request. If the user specifies that he wants to use `miniforge` and also tries to create Python 2 environment then he will get an error, so it's ok. 

As for tests: yes, it's not a must, but having tests for using `miniforge` would be hugely beneficial. See/join the - discussion in #22 about structuring the tests.

Reply:
Thanks - left a comment there.

I'd like to get this in :). Should I migrate `tests` to `tests/miniconda` and add a `tests/miniforge`? I'm happy to do the former in a separate PR.

Reply:
Yes, for now, let's have `tests/miniconda` and `tests/miniforge` with duplicated files. Not the best way, but better than no tests at all 😛 I will create another issue with designing and setting up proper test framework

Reply:
Done.